### PR TITLE
The First Task is completed

### PR DESCRIPTION
**Pull request by YehorAltShuler**
UI specifications:
https://www.figma.com/file/qMnX1YMS15Hu3vXTSvlPVH/Coffee-Delivery-App-Exploration-(Community)?node-id=1%3A2

Technical specifications:

- Create an App
- Add Bottom Navigation Bar
- Add 4 Screens with text that contains title in it

### CODE REVIEW

**Comment by YehorAltShuler** [comment]
UI specifications:
https://www.figma.com/file/qMnX1YMS15Hu3vXTSvlPVH/Coffee-Delivery-App-Exploration-(Community)?node-id=1%3A2

Technical specifications: 
- Create an App
- Add Bottom Navigation Bar
- Add 4 Screens with text that contains title in it

File: lib/main.dart [hunk 1]
```diff
@@ -7,109 +8,22 @@ void main() {
...
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
-            ),
-            Text(
-              '$_counter',
-              style: Theme.of(context).textTheme.headline4,
-            ),
-          ],
+        scaffoldBackgroundColor: const Color(0xFF201520),
```
Comment (ldemyanenko) [inline lib/main.dart:16]: Could you move all the colors you are using in the separate file/class? Try checking best practices in this

File: lib/widgets/main_navigator.dart [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        children: _screens,
+        onPageChanged: _onPageChanged,
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        showSelectedLabels: false,
+        showUnselectedLabels: false,
+        type: BottomNavigationBarType.fixed,
+        backgroundColor: Theme.of(context).colorScheme.primary,
+        selectedItemColor: Theme.of(context).colorScheme.secondary,
+        unselectedItemColor: Color(0xFF746863),
```
Comment (ldemyanenko) [inline lib/widgets/main_navigator.dart:50]: Same comment as above about colors

**Comment by ldemyanenko** [comment]
Please add one widget test that checks MainPage. It should check that this page has bottom navigation bar with 4 items in it. It should be able to click each one and check corresponding text

**Review by ldemyanenko — CHANGES_REQUESTED** [review]

File: lib/widgets/main_navigator.dart [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  @override
+  State<MainNavigator> createState() => _MainNavigatorState();
+}
+
+class _MainNavigatorState extends State<MainNavigator> {
+  PageController _pageController = PageController();
+
+  int _seletedIndex = 0;
+
+  List<Widget> _screens = [
```
Comment (ldemyanenko) [inline lib/widgets/main_navigator.dart:19]: I suggest not to put widgets in a list variable without a good reason

File: lib/widgets/main_navigator.dart [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        showSelectedLabels: false,
+        showUnselectedLabels: false,
+        type: BottomNavigationBarType.fixed,
+        backgroundColor: Theme.of(context).colorScheme.primary,
+        selectedItemColor: Theme.of(context).colorScheme.secondary,
+        unselectedItemColor: Color(0xFF746863),
+        items: [
+          BottomNavigationBarItem(icon: Icon(Icons.home), label: ''),
```
Comment (ldemyanenko) [inline lib/widgets/main_navigator.dart:52]: Use const key word where possible

File: lib/widgets/main_navigator.dart [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+          BottomNavigationBarItem(icon: Icon(Icons.shopping_cart), label: ''),
+          BottomNavigationBarItem(
+            icon: Icon(Icons.favorite),
+            label: '',
```
Comment (ldemyanenko) [inline lib/widgets/main_navigator.dart:56]: No need to set empty string there and in all other items here

**Review by ldemyanenko — CHANGES_REQUESTED** [review]

File: lib/main.dart (hunk 1 above)
Comment (YehorAltShuler) [reply lib/main.dart:17]: Made a file with constant colors.

File: lib/widgets/main_navigator.dart (hunk 3 above)
Comment (YehorAltShuler) [reply lib/widgets/main_navigator.dart]: Moved widgets from variable to children of BottomNavBar

File: lib/widgets/main_navigator.dart (hunk 4 above)
Comment (YehorAltShuler) [reply lib/widgets/main_navigator.dart]: Did it

File: lib/widgets/main_navigator.dart (hunk 5 above)
Comment (YehorAltShuler) [reply lib/widgets/main_navigator.dart]: I have to, otherwise there will be an error

**Review by ldemyanenko — APPROVED** [review]

**Comment by YehorAltShuler** [comment]
Merging Task-1 into master

**Pull request merged**